### PR TITLE
Fix zkEVM blockchains and add Linea blockchains

### PR DESCRIPTION
**Pull request by daveroga**
1. Fix zkevm blockchains
   - Mainnet
      - Blockchain: Polygon
      - NetworkId: Zkevm
      - ChainId: 1101
   - Testnet
      - Blockchain: Polygon
      - NetworkId: Cardona
      - ChainId: 2442

2. Add Linea blockchains
   - Mainnet
      - Blockchain: Linea
      - NetworkId: Main
      - ChainId: 59144
   - Testnet
      - Blockchain: Linea
      - NetworkId: Sepolia
      - ChainId: 59141

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -97,13 +101,15 @@ const blockchainNetworkMap = {
   [`${Blockchain.Polygon}:${NetworkId.Main}`]: 0b0001_0000 | 0b0000_0001,
   [`${Blockchain.Polygon}:${NetworkId.Mumbai}`]: 0b0001_0000 | 0b0000_0010,
   [`${Blockchain.Polygon}:${NetworkId.Amoy}`]: 0b0001_0000 | 0b0000_0011,
+  [`${Blockchain.Polygon}:${NetworkId.Zkevm}`]: 0b0011_0000 | 0b0000_0001,
```
Comment (volodymyr-basiuk) [inline src/constants.ts:104]: we shouldn't change Blockchain bytes, for Polygon it's `0b0001_0000`, you need to change NetworkId flag

File: src/constants.ts [hunk 2]
```diff
@@ -97,13 +101,15 @@ const blockchainNetworkMap = {
...
+  [`${Blockchain.Polygon}:${NetworkId.Cardona}`]: 0b0011_0000 | 0b0000_0010,
```
Comment (volodymyr-basiuk) [inline src/constants.ts:105]: same here

File: src/constants.ts (hunk 1 above)
Comment (volodymyr-basiuk) [reply src/constants.ts:104]: Or we shouldn't change `networkFlag` to not break anything ?
CC @vmidyllic

Comment (volodymyr-basiuk) [reply src/constants.ts:104]: we could set it like this `0b0001_0000 | 0b0010_0001,`

so first 4 bits would be the same as other Polygon and in netwrok we will keep final version of networkFlag as it was

Comment (OBrezhniev) [reply src/constants.ts:104]:
```golang
	{Blockchain: Polygon, NetworkID: Zkevm}:   0b0001_0000 | 0b0000_0100,
	{Blockchain: Polygon, NetworkID: Cardona}: 0b0001_0000 | 0b0000_0101,
```

Comment (daveroga) [reply src/constants.ts]: Fixed

**Review by volodymyr-basiuk — APPROVED** [review]

**Pull request merged**